engine/src/main/java/com/redhat/cloud/notifications/templates/Edge.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Edge" to read templates from resources/templates/Edge
+public class Edge implements EmailTemplate {
```

Comment:
Classes that implement `EmailTemplate` usually have the name of an application and not a bundle. This may need to be renamed to `FleetManagement`.